### PR TITLE
tiproxy: add traffic replay doc

### PR DESCRIPTION
**Pull request by djshow832**
### What is changed, added or deleted? (Required)

TiProxy v1.3.0 will support traffic replay as an experimental feature.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions))

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by joey-yez — APPROVED** [review]

File: tiproxy/tiproxy-command-line-flags.md [hunk 1]
```diff
@@ -120,4 +155,84 @@ level = 'warning'
...
+#### `traffic capture`
+
+`tiproxyctl traffic capture` 用于捕获流量。
+
+选项：
+
+- `--output`：（必填）指定流量文件存放的目录。
+- `--duration`：（必填）指定捕获的时长。可选单位为 `m`（分钟）、`h`（小时）或 `d`（天）。例如 `--duration=1h` 指定捕获一小时的流量。
+
+例如，以下命令连接到 TiProxy 实例 `10.0.1.10:3080`，捕获一小时的流量，并保存到 TiProxy 实例的 `/tmp/traffic` 目录下：
```
[the remark this answers is not in the document]
Comment (qiancai) [reply tiproxy/tiproxy-command-line-flags.md:169]: reason: 连续两个例如有点奇怪

File: media/tiproxy/tiproxy-traffic-replay.png
Comment (qiancai) [inline media/tiproxy/tiproxy-traffic-replay.png:1]: 这个图里，每个步骤的首字母最好能大写，production cluster 和 testing cluster 的首字母也最好能大写

Comment (djshow832) [reply media/tiproxy/tiproxy-traffic-replay.png:1]: Production cluster 还是 Production Cluster?

**Review by Oreoxmt — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**